src/interpreter/inner.rs

```diff
@@ -190,6 +200,25 @@ pub fn from_txdata<'txin>(
                 None => Err(Error::UnexpectedStackEnd),
             }
         }
+    // ** pay to taproot **//
+    } else if spk.is_v1_p2tr() {
+        if !ssig_stack.is_empty() {
+            Err(Error::NonEmptyScriptSig)
+        } else {
+            let output_key = bitcoin::XOnlyPublicKey::from_slice(&spk[2..])
+                .map_err(|_| Error::XOnlyPublicKeyParseError)?;
+            if wit_stack.len() == 1 {
+                // Key spend
+                // let sig =
+                // Tr inference to be done in future commit
+                todo!();
```

Comment:
n.b. might be better not to commit a todo! here for better bisecting sometime down the line?

Reply:
Fair point. Will change to the `unreachable!` in the commit it is introduced. In my local development, I only test that every commit compiles for stable and not for MSRV.

Reply:
Is this a comment about `todo!` not working on MSRV? Or that it might panic? If it is the latter, I think it is safe to do here because there is a new feature being added.

Reply:
not sure about that, I just meant it's nice if I want to test the git history against a taproot fragment that I can see where a bug in the interpreter gets added (e.g., like the bug I pointed out below about stack sizes would not be catchable in a bisect here).